LBH.AdultSocialCare.Api.Tests/V1/Helper/DatabaseDataHelper.cs

```diff
@@ -3,26 +3,30 @@
 using LBH.AdultSocialCare.Api.V1.AppConstants.Enums;
 using LBH.AdultSocialCare.Api.V1.Infrastructure;
 using LBH.AdultSocialCare.Api.V1.Infrastructure.Entities.Common;
+using System;
 using System.Linq;
 
 namespace LBH.AdultSocialCare.Api.Tests.V1.Helper
 {
     public static class DatabaseDataHelper
     {
```

Comment:
Why have different classes to generate packages in database?

Reply:
It's a good approach as well. No? I actually like it better

Reply:
This approach is acceptable, but it's no good for everybody to implement its version of same functionality :)

Reply:
With some naming conventions it's easier for anyone other than the original author to use it. You don't have to look through generators to find one that creates what you want. Just start typing create and intellisense helps find all options. If missing add in there and next person can use it. Just thinking

Reply:
Just the way someone tries doing `ToDomain` or `ToResponse` before creating

Reply:
ok, assumed that it's easier to navigate in smaller classes, but let's join then, it's absolutely not an issue :)

Will join, anything else?

Reply:
None from me. Maybe if you have something else for me

Reply:
Nope, will approve now and join generators in next PR